core/modules/installer/installer.module

```diff
@@ -0,0 +1,510 @@
+<?php
+/**
+ * @file
+ * Handles installation and updates of contributed projects.
+ */
+
+/**
+ * Implements hook_permission().
+ */
+function installer_permission() {
+  return array(
+    'use installer browser' => array(
+      'title' => t('use installer browser'),
```

Comment:
Permission titles are usually sentence case, so at least capitalize "Use" here. Though internally "installer browser" I think is fine, it doesn't roll right off the tongue for an end-user feature. Could we change this up to "Access installer interface" or similar?